src/components/AllJobLogsView/AllJobLogsView.js

```diff
@@ -189,7 +184,7 @@ export const AllJobLogsViewComponent = ({
         />
         <ResetButton
           id="reset-export-filters"
-          disabled={checkObjectValues(filters)}
+          disabled={!location.search || isResetButtonDisabled}
```

Comment:
!location search can be added to isResetButtonDisabled

Reply:
Thanks @vashjs , I made that changes